tensorflow/lite/micro/kernels/softmax_common.cc

```diff
@@ -28,11 +28,48 @@ namespace {
 // Softmax parameter data that persists in user_data
 const int kInt16LUTArraySize = 513;
 
+}  // namespace
+
 TfLiteStatus CalculateSoftmaxParams(TfLiteContext* context,
                                     const TfLiteTensor* input,
                                     TfLiteTensor* output,
                                     const TfLiteSoftmaxParams* params,
                                     SoftmaxParams* op_data) {
+  // Only allocate LUTs for KTfLiteInt16 data type
```

Comment:
Could line 38 to line 71 be extracted as a separate function such as InitializeLutForInt16? This is a bit long and the function name will make it more readable.

Reply:
Done